addon/components/ember-th/component.js

```diff
@@ -115,6 +117,10 @@ export default BaseTableCell.extend({
     this._hammer = hammer;
   },
 
+  didRender() {
+    this.get('columnMeta').registerElement(this.element);
+  },
```

Comment:
Upon initial render, [`didRender` gets called immediately following `didInsertElement`](https://guides.emberjs.com/v3.12.0/components/the-component-lifecycle/). If we're going to execute this line of code in `didRender`, then I don't believe that we should *also* be calling it above in `didInsertElement`.

Reply:
Thanks for catching this.

Reply:
Why is this being moved to `didRender`?